config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
+  get 'users/new'
   root "static_pages#home"
-  get "static_pages/home"
-  get "static_pages/help"
-  get "static_pages/about"
+  get "/help",    to: "static_pages#help", as: "herf"
```

Comment:
```suggestion
  get "/help", to: "static_pages#help", as: "herf"
```
Sau dấu , cách 1 space thôi